### PR TITLE
invoke-signer draft 0

### PR DESCRIPTION
**Pull request by zmitton**
Im looking into defining a spec to specifically invoke a user’s preferred wallet app for signing (a tx or arbitrary data)

I think it's time for a standard that can accommodate the user to use a hardware wallet from a desktop browser dapp or open a separate wallet app from a mobile browser dapp etc...

Discussions to here please 👉: https://github.com/ethereumclassic/ECIPs/issues/147

### CODE REVIEW

File: _specs/invoke-signer.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+## Standardize OS Level Management of Signature Handling
+
+    ECLIP: <pending>
```
Comment (soc1c) [inline _specs/invoke-signer.md]:
```suggestion
    ECIP: 1067
```

Comment (zmitton) [reply _specs/invoke-signer.md]: Thanks, just going to wait 24 hours to see if it gets an EIP number instead. If not, I will commit this one

**Comment by bobsummerwill** [comment]
How would this relate to WalletConnect?

File: _specs/invoke-signer.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    Status:  DRAFT
-    Type: STANDARDS TRACK
-    Created: 2019-02-07
+---
+lang: en
+ecip: 1067
+title: Standardize OS Level Management of Signature Handling
+status: Draft
+type: Interface
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/58
```
Comment (zmitton) [inline _specs/invoke-signer.md]: thank you, but you've put the wrong discussions-to link @sorpaas 
```suggestion
discussions-to: https://github.com/ethereumclassic/ECIPs/issues/147
```

**Pull request merged**